scipy/spatial/ckdtree.pyx

```diff
@@ -1448,6 +1469,26 @@ cdef class cKDTree:
             format. If a dict is returned the keys are (i,j) tuples of indices.
             If output_type is 'ndarray' a record array with fields 'i', 'j',
             and 'v' is returned,
+
+        Examples
+        --------
+        You can compute a sparse distance matrix between two kd-trees:
+
+        >>> import numpy as np
+        >>> from scipy.spatial import cKDTree
+        >>> np.random.seed(21701)
+        >>> points1 = np.random.random((5, 2))
+        >>> points2 = np.random.random((5, 2))
+        >>> kd_tree1 = cKDTree(points1)
+        >>> kd_tree2 = cKDTree(points2)
+        >>> sdm = kd_tree1.sparse_distance_matrix(kd_tree2, 1.0)
```

Comment:
Perhaps reduce  the radius to 0.2 so we can see the sparsity in the matrix?

Reply:
Maybe even compare to `spatial.distance_matrix` to show how the sparsity matches with the distances above the threshold.

Reply:
good idea. I fixed it. PTAL.